### PR TITLE
Don't create empty exception file on UWP

### PR DESCRIPTION
**Pull request by guperrot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the UWP implementation?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Don't create empty exception file on UWP.

## Related PRs or issues

[AB#63628](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/63628)

### CODE REVIEW

File: SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs [hunk 1]
```diff
@@ -236,27 +236,29 @@ private Task ProcessPendingErrorsAsync()
                         }
                         continue;
                     }
-                    var report = BuildErrorReport(log);
-                    if (report == null)
+
+                    // The error report cannot be built if any of those fields are null.
+                    if (log.Device == null || log.AppLaunchTimestamp == null || log.Timestamp == null)
```
Comment (clpolet) [inline SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs]: Would calling Validate() on the log make sense here?

Comment (jaelim-ms) [reply SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs]: It is a generated data model I guess.

Comment (jaelim-ms) [reply SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs]: [Here](https://github.com/microsoft/appcenter-sdk-dotnet/blob/develop/scripts/generated/Models/Log.cs)
In the generated model, `Device` is the only one nonnull.

Comment (jaelim-ms) [reply SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs]: I also had a same doubt but I believed my assumption is true. 😄

**Pull request merged**